components/admin_console/license_settings/license_settings.jsx

```diff
@@ -175,13 +175,13 @@ export default class LicenseSettings extends React.Component {
             // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
             edition = (
                 <p>
-                    {'Mattermost Enterprise Edition. Unlock enterprise features in this software through the purchase of a subscription from '}
+                    {'Mattermost Enterprise Edition. Unlock enterprise features in this software by starting a trial subscription at '}
```

Comment:
shouldn't it be `Mattermost Team Edition. Unlock enterprise...`?

Reply:
@etoaster & @jespino Recommend adjusting to make it more clear: "Mattermost Enterprise Edition. A license is required to unlock enterprise features in this software. Start a trial subscription at"

Reply:
Changed the language to what @wiersgallak suggested.